src/shared/chainConfig/sepolia.ts

```diff
@@ -0,0 +1,107 @@
+import { AxiomV2CircuitConstant } from "@axiom-crypto/tools";
+import { Endpoints, Path } from "../endpoints";
+
+export let versionDataSepolia: any = {
+  v0: {
```

Comment:
unfortunately, since each of the files in chainConfig follows a specific template, i'd rather keep this as-is for now and not risk anything breaking anything. it's definitely something i want to refactor later though.